frontend/src/services/product.service.js

```diff
@@ -0,0 +1,14 @@
+import httpCommon from "../http-common";
+class ProductDataService {
+    getHighlighted() {
+      return  httpCommon.get("/productRandom");
+    }
+    getNewestOrLastChance(offer,start,count){
+      if(offer == 1){
+        return httpCommon.get("/newest?start="+start+"&count="+count)
+      }else if(offer == 2){
+        return httpCommon.get("/lastChance?start="+start+"&count="+count)
+      }
+    }
```

Comment:
Maybe you can write it like this :
return httpCommon.get("/" + (offer === 1 ? "newest": "lastChance") +"?start="+start+"&count="+count);